comp/core/tagger/global.go

```diff
@@ -20,15 +17,23 @@ import (
 var (
 	// globalTagger is the global tagger instance backing the global Tag functions
 	// // TODO(components) (tagger): globalTagger is a legacy global variable but still in use, to be eliminated
-	globalTagger *TaggerClient
+	globalTagger Component
```

Comment:
we had a pointer before, and now this is not a pointer is this intended ?

Reply:
Yes. We had to modify this to accept the tagger component interface so we can have a more unified experience when passing tagger types around.

Reply:
okay perfect